app/models/journeys/additional_payments_for_teaching.rb

```diff
@@ -9,5 +9,8 @@ module AdditionalPaymentsForTeaching
     VIEW_PATH = "additional_payments"
     I18N_NAMESPACE = "additional_payments"
     POLICIES = [Policies::EarlyCareerPayments, Policies::LevellingUpPremiumPayments]
+    FORMS = {
+      "poor-performance" => ::PoorPerformanceForm
+    }.freeze
```

Comment:
Suggestion: going with Rails' 'convention over configuration' principle perhaps the form class name could be inferred from the slug name.

Reply:
It's worth postponing this to later so this PR can unblock @rjlynch's work. It's definitely on the radar